modules/snooker/src/main/ts/ephox/snooker/selection/SelectionTypes.ts

```diff
@@ -0,0 +1,38 @@
+import { Element as DomElement } from '@ephox/dom-globals';
+import { Adt } from '@ephox/katamari';
+import { Element } from '@ephox/sugar';
+
+export interface SelectionType {
+  fold: <T>(
+    none: () => T,
+    multiple: (elements: Element<DomElement>[]) => T,
+    single: (elements: Element<DomElement>) => T,
```

Comment:
Typo in the variable name here and below, as this should just be `element`.

Reply:
Missed one, will fix